lib/events/auditlog.go

```diff
@@ -348,6 +349,7 @@ func (l *AuditLog) UploadSessionRecording(r SessionRecording) error {
 	}
 	l.log.WithFields(log.Fields{"duration": time.Since(start), "session-id": r.SessionID}).Debugf("Session upload completed.")
 	return l.EmitAuditEventLegacy(SessionUploadE, EventFields{
+		EventID:        uuid.New(),
```

Comment:
Do you know under what circumstances this is executed? I didn't think UploadSessionRecording was ever used anymore.